### PR TITLE
Create 2019-11-16-robot-framework-for-network-testing.md

### PR DESCRIPTION
**Pull request by lara29**
Blog for the Robot framework tutorial

### CODE REVIEW

File: content/post/2019-11-16-robot-framework-for-network-testing.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+date: 2018-11-16T00:00:00-00:00
+description: "Automated testing made easy and efficient with Robot framework"
+featured_image: "/images/nrelabs.png"
+tags: [verification, testing]
+title: "Using Robot Framework for Automated Testing"
+show_reading_time: true
+authors:
+  name: Saim Khan
```
Comment (Mierdin) [inline content/post/2019-11-16-robot-framework-for-network-testing.md]: I'll review the rest later, but for now, I'll mention that the current template doesn't support multiple authors. however, this is an easy fix, and I'll make the necessary changes. In the meantime, please convert this to a list of dicts - in the current format, the key `Saim Khan` would get overwritten.

I.e.:

```
- name: Saim Khan
  link: https://github.com/saimkhan92
- name: Lakshmi Rajan
  link: https://github.com/lara29
```

**Comment by Mierdin** [comment]
FYI - I recently added support for multiple authors in https://github.com/nre-learning/blog/commit/91ceaedfb00a0d3db55ff9bcd9598262e085adad - please convert your `author` key to `authors`, and change the value to a list of dicts, as shown in that commit for existing lessons.

File: content/post/2019-11-16-robot-framework-for-network-testing.md (hunk 1 above)
Comment (lara29) [reply content/post/2019-11-16-robot-framework-for-network-testing.md]: Thanks Matt, I've changed the authors to list of dicts and committed

File: content/post/2019-11-16-robot-framework-for-network-testing.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+date: 2018-11-16T00:00:00-00:00
+description: "Automated testing made easy and efficient with Robot framework"
```
Comment (Mierdin) [inline content/post/2019-11-16-robot-framework-for-network-testing.md]: Please add draft tag via https://gohugo.io/getting-started/usage/

Comment (lara29) [reply content/post/2019-11-16-robot-framework-for-network-testing.md]: Added the draft tag in the latest commit

Comment (lara29) [reply content/post/2019-11-16-robot-framework-for-network-testing.md]: @Mierdin I've added draft: true to the top section of the blog. Is this what you meant?

**Comment by lara29** [comment]
Thanks @Mierdin for reviewing and merging the blog. Sure, I will check it once you have edited it.

**Comment by Mierdin** [comment]
@lara29 FYI https://github.com/nre-learning/nre-blog/pull/5

**Pull request merged**